src/main/java/frc/robot/subsystems/Indications.java

```diff
@@ -49,31 +51,59 @@ public void flashing(LEDSubStrip section, Color color, int interval) {
     for (int i = 0; i <= section.getLength(); i++) {
       // turns the LEDs onn if the interval is one
       if ((interval % 2) == 1) {
-        section.setLED(i, color);
+        armLEDsBuffer.setLED(i, color);
       } else {
         // sets the LEDs off
-        section.setRGB(i, 0, 0, 0);
+        armLEDsBuffer.setLED(i, Color.kBlack);
       }
     }
   }
 
-  public void runway(LEDSubStrip section, Color color, int interval) {
+  public void runaway(LEDSubStrip section, Color color, int interval) {
     // finds many LEDS are in the section
     int sectionSize = section.getLength();
 
     for (int i = 0; i <= section.getLength(); i++) {
       // finds where is the sequence the "chasing" LED is
-      if ((i % sectionSize) == interval) {
+      if (i % sectionSize >= (interval + sectionSize) / kIndications.maxTick
+          && (i % sectionSize) + 1 < (interval + sectionSize) / kIndications.maxTick) {
         armLEDsBuffer.setLED(i, color);
       } else {
         armLEDsBuffer.setRGB(i, 0, 0, 0);
       }
     }
   }
 
+  private void flashingRainbow(LEDSubStrip section, int interval) {
+    // Altered version of the rainbow script from the first documentation
+    for (var i = 0; i < section.getLength(); i++) {
+      if ((interval / 30) % 2 == 0) {
+        final var hue =
+            (interval * 180 / (kIndications.maxTick) + (i * 180 / section.getLength())) % 180;
+        armLEDsBuffer.setHSV(i, hue, 255, 128);
+      } else {
+        armLEDsBuffer.setLED(i, Color.kBlack);
+      }
+    }
+  }
+
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
+
+    // changes tick by one and than
+    tick++;
+
+    if (tick >= kIndications.maxTick) {
+      tick = 0;
+    }
```

Comment:
```suggestion

    // increment tick
    tick = (tick + 1) % kIndications.maxTick;
```
This is a more concise way you can achieve the same wrapping count behavior.